### PR TITLE
Implement `encode(to:)` method of `Event` (fixes #252)

### LINKED ISSUE #252: EventObject is not actually Encodable
The `encode(to encoder: Encoder)` method of `EventObject` is empty.

This is bad!
Either the object conforms to `Encodable` and should encode properly, or it’s not and it should not conform to `Encodable`. At the very least it should throw an error if for some reason it is not possible to have `Encodable` conformance but said conformance is required for external reasons.

My use-case: I send Stripe events to be processed by in a queue (as one should), and with the current implementation I just receive an empty object in my job.

### PR DESCRIPTION
**Pull request by Frizlab**

### CODE REVIEW

File: Tests/StripeKitTests/EventTests.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import Foundation
```
Comment (Andrewangeta) [inline Tests/StripeKitTests/EventTests.swift:1]: We can delete this test case. No specific behavior outside of Codable itself :)

File: Sources/StripeKit/Core Resources/Events/Event.swift [hunk 2]
```diff
@@ -232,7 +232,112 @@ public enum EventObject: Codable {
         }
     }
     
-    public func encode(to encoder: Encoder) throws { }
+    public func encode(to encoder: Encoder) throws {
+        switch self {
+        case .account(let connectAccount):
```
Comment (Andrewangeta) [inline Sources/StripeKit/Core Resources/Events/Event.swift:237]: Also this is just happenstance but #254 added a new object that an `Event` should decode. Can you update the encode decode method to handle that? I think the `decode` was already added.

**Review by Andrewangeta — CHANGES_REQUESTED** [review]

**Comment by Frizlab** [comment]
Done and done!

**Review by Andrewangeta — APPROVED** [review]

**Comment by Andrewangeta** [comment]
Thank you sir :) will merge a tag a release

**Pull request merged**